### PR TITLE
reimport library content for okd/ocp tag changes for operator; remove…

### PR DESCRIPTION
**Pull request by gabemontero**
… community/official dir struct to seamlessly get imagestreams and templates in same fashion

@openshift/sig-developer-experience ptal

### CODE REVIEW

File: pkg/stub/handler.go [hunk 1]
```diff
@@ -497,18 +496,17 @@ func (h *Handler) GetBaseDir(arch string, opcfg *v1alpha1.SamplesResource) (dir
...
 		default:
 			err = fmt.Errorf("invalid install type %s specified, should be rhel or centos", string(opcfg.Spec.InstallType))
 		}
 	case ppc:
 		switch opcfg.Spec.InstallType {
 		case v1alpha1.RHELSamplesDistribution:
-			dir = ppc64OfficialContentRootDir
 		case v1alpha1.CentosSamplesDistribution:
-			dir = ppc64CommunityContentRootDir
+			dir = ppc64OCPContentRootDir
```
[the remark this answers is not in the document]
Comment (gabemontero) [reply pkg/stub/handler.go]: Hmm, ok based on https://github.com/openshift/library/pull/114#issuecomment-421446608 and `There is currently no content tagged for okd+ppc64le`, I took that as specifying centos for ppc64 as unsupported, and fix things / force them down the rhel/ocp path regardless.

Would you rather I return an error if they specify centos+ppc ?

[the remark this answers is not in the document]
Comment (bparees) [reply pkg/stub/handler.go]: i'd say either error, or simply install nothing for now (i'm guessing someday we will have content tagged for okd+ppc64).

but also the way you have this case currently structured, dir isn't going to get set at all for InstallType==v1alpha1.RHELSamplesDistribution, right?  (the case statement is empty for it).

[the remark this answers is not in the document]
Comment (gabemontero) [reply pkg/stub/handler.go]: duh !! :-) ... getting my java and golang confused .... I needed a `fallthrough` to achieve what I intended, which was to have *both* rhel and centos result ind dir being set to ppc64OCP

I'll fix the case stmt, and go with an error for ppc/centos

**Comment by gabemontero** [comment]
@bparees pushed the update re: the arch case stmt

**Comment by bparees** [comment]
squash and lgtm

**Pull request merged**